test/unit/spec/common/jobCommChannel-spec.js

```diff
@@ -86,887 +86,962 @@ define([
             expect(comm.initCommChannel).toBeDefined();
         });
 
-        it('Should initialize correctly in the base case', () => {
-            const comm = new JobCommChannel();
-            return comm.initCommChannel().then(() => {
-                expect(comm.comm).not.toBeNull();
+        describe('start up', () => {
```

Comment:
Big diffs due to organising the tests into some `describe` sections. There are no content changes until right near the end of the file.

Reply:
To be totally honest, I'm just skipping down to the new stuff at the bottom...